src/styles/_themes.scss

```diff
@@ -14,6 +14,12 @@ $themes: (
     headerColor: $black,
```

Comment:
Suppose the designed wants another color instead of $black. The chance that you would fail to replace all $blacks is high. So better have in _variables.scss:

$black: '#000000';
$darkThemeThemeColor: $black;
$lightThemeThemeColor: $green;
And then import the $darkThemeThemeColor in any other file. Then you will only need to replace it in one place.

Please, go through all your styles and refactor.

Reply:
refactor some styles